recipe/meta.yaml

```diff
@@ -59,23 +68,24 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }}                        # [cuda_compiler_version != "None"]
+    - {{ compiler('cuda') }}                        # [(cuda_compiler_version or "").startswith("12")]
     - cmake
     - git
     - ninja
     - pkgconfig
   host:
     # NOTE: Without cuda-version, we are installing cuda-toolkit 11.8 instead of 11.2!
-    - cuda-version     {{ cuda_compiler_version }}  # [cuda_compiler_version != "None" and not (cuda_compiler_version or "").startswith("12")]
+    - cudatoolkit      {{ cuda_compiler_version }}  # [cuda_compiler_version != "None" and not (cuda_compiler_version or "").startswith("12")]
```

Comment:
Same here, `cuda-version` would already pull `cudatoolkit`.

Reply:
not for cuda != 12

Reply:
hmm need to look at this closer, thanks for the catch

Reply:
I don't really know why we changed this...

Reply:
yeah this change stays the same because it handles v11.8. We don't have a cuda-version for that, whereas conda-forge does. But cuda-version needs to be added to run for v12.x

Reply:
There is no cudatoolkit for v12

Reply:
Umm, this comes back to "what's conda-version used for...". I feel like we (or at least I) are either missing something or the upstream conda-forge recipe is wrong...

Reply:
I agree, see other thread in this PR